homeassistant/components/abode/config_flow.py

```diff
@@ -112,9 +112,6 @@ async def async_step_user(
         self, user_input: dict[str, Any] | None = None
     ) -> ConfigFlowResult:
         """Handle a flow initialized by the user."""
-        if self._async_current_entries():
```

Comment:
Do we test this in the config flow tests?

Reply:
https://github.com/home-assistant/core/blob/2da73ea0687c0a7be8fa43bd71a872937aee111c/tests/components/abode/test_config_flow.py#L24-L36